app/views/modals/set-home-page-modal.html

```diff
@@ -0,0 +1,48 @@
+<div class="modal-resource-action">
+  <div class="modal-header">
+    <h1>Set Home Page</h1>
+  </div>
+  <div class="modal-body indent">
+    <p>
+      <div class="title">Set your preferred home page.  This is the page you will see at login.</div>
```

Comment:
It's not necessarily the page you see when you login. It's the page if there's no URL path. We need to let users link directly to other pages.

Reply:
@ncameronbritt, could you please re-wordsmith this?  -thanks

Reply:
I feel like just removing that second sentence covers it. I believe that users would understand what a home page means.